subcommands/waves/init.go

```diff
@@ -28,9 +29,15 @@ This command only initializes a wave, but does not provision its updates to devi
 Use a "fioctl wave rollout <wave> <group>" to trigger updates of this wave to a device group.
 Use a "fioctl wave complete <wave>" to update all devices (make it globally available).
 Use a "fioctl wave cancel <wave> to cancel a wave (make it no longer available).`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			confirmWaveInitProceed()
+		},
```

Comment:
Please, simply add the dialog into `doWaveInit`, don't make a control flow more complex to grasp.